steps/split.py

```diff
@@ -19,4 +19,4 @@ def create_dataset_filter(dataset: DataFrame) -> Series(bool):
     """
     # FIXME::OPTIONAL: implement post-split processing on the dataframes, such as data cleaning.
```

Comment:
Can we also update the comment since you are at it?

Reply:
What part of the comment? "post-split processing" -> "post-split filtering" or something else

Reply:
Yep.